packages/frontend/electron/src/main/main-window.ts

```diff
@@ -127,7 +127,11 @@ async function createWindow(additionalArguments: string[]) {
       //   - all browser windows will capture the "close" event
       //   - the hidden window will close all windows
       //   - "window-all-closed" event will be emitted and eventually quit the app
-      browserWindow.hide();
+      if (browserWindow.isFullScreen()) {
+        browserWindow.setFullScreen(false);
```

Comment:
can we do both `setFullscreen(false)`  and `hide()`

Reply:
It seems setting full screen to false will trigger an animation. During animation transitioning period, `hide` will not work.

Reply:
https://github.com/electron/electron/issues/6033#issuecomment-242023295

seems to be a solution

[![confused](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/9403235f-2f1e-48f1-a6fc-2a3a04a41ea4/confused.gif)](https://app.graphite.dev/settings/meme-library?org=toeverything)